app/services/foreman_salt/fact_parser.rb

```diff
@@ -0,0 +1,78 @@
+module ForemanSalt
+  class FactParser < ::PuppetFactParser
```

Comment:
How Salt handles operatingsystems is basically identical to puppet, and I didn't want to repeat all that logic.

But is it that useful to anybody else?  Chef or some other config mangaement system may use entirely different fact names and values.